test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/q_and_mq_as_uri_param_for_metadata.test

```diff
@@ -0,0 +1,454 @@
+                        "value": 2
+                    }
+                }, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "id": "E2", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+08. Modify the subscription to have q=A1 AND mq=A1.M1
+=====================================================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+09. Dump accumulator, see E1 and E2, then reset accumulator
+===========================================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 256
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {
+                    "M1": {
+                        "type": "none", 
+                        "value": 1
+                    }
+                }, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "id": "E1", 
+            "type": "none"
+        }, 
+        {
+            "A1": {
+                "metadata": {
+                    "M1": {
+                        "type": "none", 
+                        "value": 2
+                    }
+                }, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "id": "E2", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+10. Update E3, adding M1 to A1
+==============================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+11. Dump accumulator, see E3
+============================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 186
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {
+                    "M1": {
+                        "type": "none", 
+                        "value": 3
+                    }, 
+                    "M2": {
+                        "type": "none", 
+                        "value": 3
+                    }
+                }, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "id": "E3", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
+accumulatorStop
```

Comment:
I'd suggest to add a couple more .test:

* Test "FIWARE:StringQuery::Metadata" in /v1/queryContext scope
* Test "FIWARE::StringQuery::Metadata" in /v2/op/query scope